action_server/frontend/src/views/actions/components/ActionRun.tsx

```diff
@@ -244,82 +229,70 @@ export const ActionRun: FC<Props> = ({ action, actionPackage }) => {
             switch (item.property.type) {
               case 'boolean':
                 return (
-                  <Item title={item.title} name={item.name} key={item.name}>
-                    <Checkbox
-                      key={`${title}-${item.name}-${index}`}
-                      label={title}
-                      description={item.property.description}
-                      checked={typeof item.value === 'boolean' && item.value}
-                      required={item.required}
-                      onChange={(e) => handleInputChange(e.target.checked, index)}
-                    />
-                  </Item>
+                  <Checkbox
```

Comment:
Note: the issue fixed here is that `item.title` isn't really defined, but after fixing then it was shown twice (one in `<Item title` and another in `<Checkbox label`, `<Input.label`, etc).

So, I opted to just remove the `Item` part.